src/mgmt/srv_target.c

```diff
@@ -276,17 +276,16 @@ ds_mgmt_tgt_setup(void)
 		D_GOTO(err_newborns, rc = -DER_NOMEM);
 
 	stored_mode = umask(0);
-	mode = S_IRWXU | S_IRWXG | S_IRWXO;
 	/** create NEWBORNS directory if it does not exist already */
-	rc = mkdir(newborns_path, mode);
+	rc = mkdir(newborns_path, S_IRWXU);
```

Comment:
is different users allowed to use the same DAOS backend (like two users start their daos servers and share to use the same /mnt/daos)? if so 700 is not enough?

Reply:
Hmm, these are on the server side. I tend to think that only one user(admin) will start the service job. 
Different users to start the daos servers under the same directory is only for development purpose.